zerver/lib/actions.py

```diff
@@ -4177,9 +4193,13 @@ def do_update_message_flags(user_profile: UserProfile,
     send_event(user_profile.realm, event, [user_profile.id])
 
     if flag == "read" and operation == "add":
+        event_time = timezone_now()
         do_clear_mobile_push_notifications_for_ids(user_profile, messages)
 
-    statsd.incr("flags.%s.%s" % (flag, operation), count)
+        do_increment_logging_stat(user_profile, COUNT_STATS['messages_read::hour'],
```

Comment:
I am not completely sure that we really want to remove statsd logging especially here where we store the logs other operations too. Based on what you mentioned [https://github.com/zulip/zulip/issues/15167](https://github.com/zulip/zulip/issues/15167)

Reply:
Yes, we want to remove `statsd`; we haven't used statsd in years; we'd just left the code here as a marker for where we'd want to do analytics on reading messages.